src/file_16.py

```diff
@@ -0,0 +1,30 @@
+# task_16
+
+BOARD_SIZE = 8
+
+
+def under_attack(col, queens):
```

Comment:
here you have 2 empty lines between constant and code, earlier you put 0 lines. Styling should be consistent.

Reply:
We use 2 empty lines because we have function "def ... " after constant. If we use one line, we'll get the next problem "PEP 8: E302 expected 2 blank lines, found 1" in PyCharm.